sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -739,8 +739,7 @@
     },
     "/twins/{id}": {
       "get": {
-        "summary": "Gets a device twin.",
-        "description": "Gets a device twin. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "Gets the device state information. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information.",
```

Comment:
Shouldn't this just be "Gets the device twin"? the description of the 200 status below seems to fit with "twin" rather than "state"

Reply:
It makes more sense to define a "Twin" as a device/module state within the return type of this method.

Reply:
Sure